tests/test_remesh.py

```diff
@@ -16,7 +16,7 @@
 _MESH_DIM = 3
 _BOUNDARY_DIM = 2
 
-USE_MMG = find_executable("mmg3d_O3") is not None
+USE_MMG = shutil.which("mmg3d_O3") is not None

```

Comment:
This change is somehow related to updating to python 3.12?

Reply:
distutils have been deprecated with python 3.12